modules/optflow/src/rlof/berlof_invoker.hpp

```diff
@@ -2103,6 +1850,640 @@ class TrackerInvoker  : public cv::ParallelLoopBody
+                                    vAxy = v_muladd(fx, fy, vAxy);
+                                    vAxx = v_muladd(fx, fx, vAxx);
+
+                                    // sumIx und sumIy
+                                    vsumIx += fx;
+                                    vsumIy += fy;
+
+                                    vsumW1 += vI_ps * fx;
+                                    vsumW2 += vI_ps * fy;
+
+                                    // sumI
+                                    vsumI += vI_ps;
+
+                                    // sumDI
+                                    vsumDI += vI_ps * vI_ps;
+
+                                    v01 = v_reinterpret_as_s16(v_interleave_pairs(v_reinterpret_as_s32(v_interleave_pairs(vIxy_1))));
+                                    v_expand(v01, t1, t0);
+                                    vI_ps = v_cvt_f32(vI1);
+
+                                    fy = v_cvt_f32(t0);
+                                    fx = v_cvt_f32(t1);
+
+                                    // A11 - A22
+                                    vAyy = v_muladd(fy, fy, vAyy);
+                                    vAxy = v_muladd(fx, fy, vAxy);
+                                    vAxx = v_muladd(fx, fx, vAxx);
+
+                                    // sumIx und sumIy
+                                    vsumIx += fx;
+                                    vsumIy += fy;
+
+                                    vsumW1 += vI_ps * fx;
+                                    vsumW2 += vI_ps * fy;
+
+                                    // sumI
+                                    vsumI += vI_ps;
+
+                                    // sumDI
+                                    vsumDI += vI_ps * vI_ps;
+                                }
+                            }
+
+#else
+                            for (int x = 0; x < _winSize.width*cn; x++, dIptr += 2)
+                            {
+                                if (maskPtr[x] == 0)
+                                    continue;
+
+                                short ixval = dIptr[0];
+                                short iyval = dIptr[1];
+                                int illValue = static_cast<int>(Iptr[x] * gainVec.x + gainVec.y - Iptr[x]);
+
+                                int It[4] = { (Jptr1[x + cn] << 5) + illValue,
+                                                (Jptr[x + cn] << 5) + illValue,
+                                                (Jptr1[x] << 5) + illValue,
+                                                (Jptr[x] << 5) + illValue };
+
+
+                                int J_val = CV_DESCALE(Jptr[x] * iw00 + Jptr[x + cn] * iw01 +
+                                    Jptr1[x] * iw10 + Jptr1[x + cn] * iw11,
+                                    W_BITS1 - 5);
+
+                                int diff = J_val + illValue;
+                                int abss = (diff < 0) ? -diff : diff;
```

Comment:
> abss

Looks like this variable is not used here (see "Linux Debug" compiler warning). Is it expected?

Reply:
You are right. Removed line 2218 and 2219.

Reply:
The [next](http://pullrequest.opencv.org/buildbot/builders/precommit-contrib_linux64_no_opt/builds/10312) is "J_val":

> berlof_invoker.hpp:2213:37: warning: unused variable 'J_val'

Reply:
Thank you! I am trying to setup my visual studio, so that it shows me such errors too. Hopfully this was the last unused value

Reply:
[Linux x64 Debug](http://pullrequest.opencv.org/#/summary/contrib):

> berlof_invoker.hpp:1971:40: warning: unused variable 'W_BITS1'